### DISCUSSION TITLE
The referenced component '..\winterop\winterop.vcxproj' could not be found

### DISCUSSION THREAD

**Discussion by form2t** [cake-build/cake#3392] [Q&A]
Hi, all.
I download wix3-wix38rtm.zip (release) and wheh i start wix.sln an warning "The referenced component '..\winterop\winterop.vcxproj' could not be found".  
When i build solution an error
The command ""D:\Documents\Downloads\wix3-wix38rtm\build\ship\x86\xsdgen.exe" "D:\Documents\Downloads\wix3-wix38rtm\src\tools\wix\Xsd\wix.xsd" "D:\Documents\Downloads\wix3-wix38rtm\build\obj\ship\x86\Wix\wix.cs" Microsoft.Tools.WindowsInstallerXml.Serialize " exited with code 9009.
how to fix it, thanks.

**Comment by patriksvensson** [comment]
I suspect that this message was meant for the WiX-project which can be found at https://github.com/wixtoolset/.

**Comment by form2t** [comment]
yes
